client/src/components/Shared/Form/index.js

```diff
@@ -18,68 +17,77 @@ import {
+        </StyledImgCon>
 
+        <StyledInformation>PIECE INFORMATION</StyledInformation>
+        <StyledNotic>Please verify the items marked* </StyledNotic>
 
-    <StyledDiv>
 
-      <StyledLabels>
+        <StyledDiv>
 
-        <StyledLabel> Item Type* </StyledLabel>
-        <StyledLabel> Color* </StyledLabel>
-        <StyledLabel> Brand </StyledLabel>
-        <StyledLabel> Other </StyledLabel>
-        <StyledLabel> Condition </StyledLabel>
-        <StyledLabel> Label size </StyledLabel>
-        <StyledLabel> Age </StyledLabel>
-        <StyledLabel> purchase price </StyledLabel>
+          <StyledLabels>
 
-      </StyledLabels>
+            <StyledLabel> Item Type* </StyledLabel>
+            <StyledLabel> Color* </StyledLabel>
+            <StyledLabel> Brand </StyledLabel>
+            <StyledLabel> Other </StyledLabel>
+            <StyledLabel> Condition </StyledLabel>
+            <StyledLabel> Label size </StyledLabel>
+            <StyledLabel> Age </StyledLabel>
+            <StyledLabel> purchase price </StyledLabel>
 
-      <StyledItem>
+          </StyledLabels>
 
-        <StyledSelect>
-          <StyledOption value="T-shirt">T-shirt</StyledOption>
-        </StyledSelect>
+          <StyledItem>
 
-        <StyledSelect>
-          <StyledOption value="white">white</StyledOption>
-        </StyledSelect>
+            <StyledSelect>
+              <StyledOption value="T-shirt">T-shirt</StyledOption>
+            </StyledSelect>
 
-        <StyledSelect>
-          <StyledOption value="Nike">Nike</StyledOption>
-        </StyledSelect>
+            <StyledSelect>
+              <StyledOption value="white">white</StyledOption>
+            </StyledSelect>
 
-        <StyledInput type="text" name="brand" placeholder="brand" />
+            <StyledSelect>
+              <StyledOption value="Nike">Nike</StyledOption>
+            </StyledSelect>
 
-        <StyledSelect>
-          <StyledOption value="Nike">Nike</StyledOption>
-        </StyledSelect>
+            <StyledInput type="text" name="brand" placeholder="brand" />
 
-        <StyledSelect>
-          <StyledOption value="Nike">Nike</StyledOption>
-        </StyledSelect>
+            <StyledSelect>
+              <StyledOption value="Nike">Nike</StyledOption>
+            </StyledSelect>
 
-        <StyledSelect>
-          <StyledOption value="Nike">Nike</StyledOption>
-        </StyledSelect>
+            <StyledSelect>
+              <StyledOption value="Nike">Nike</StyledOption>
+            </StyledSelect>
 
-        <StyledInput type="text" name="price" placeholder="price" />
+            <StyledSelect>
+              <StyledOption value="Nike">Nike</StyledOption>
+            </StyledSelect>
 
-      </StyledItem>
+            <StyledInput type="text" name="price" placeholder="price" />
 
-    </StyledDiv>
+          </StyledItem>
 
-    <StyledTextarea name="extra" placeholder="More  . e.g. What do you love about it?" />
+        </StyledDiv>
```

Comment:
I would call this a styled container instead.

Reply:
it's text area